From fc5d0ad3eec296edfb9f4394e5cc50b3bef8d6d8 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Thu, 19 Jun 2025 10:15:35 +0200
Subject: [PATCH] core: when removing a job from a transaction, include in
 structured log message which

(cherry picked from commit becbd2ec4e00aafb656ebc4b2977c1f62adc2ee8)

Related: RHEL-106260
---
 src/core/transaction.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/core/transaction.c b/src/core/transaction.c
index cb5c59de5c..5ad43ea378 100644
--- a/src/core/transaction.c
+++ b/src/core/transaction.c
@@ -424,6 +424,8 @@ static int transaction_verify_order_one(Transaction *tr, Job *j, Job *from, unsi
                                                     delete->unit->id, job_type_to_string(delete->type),
                                                     j->unit->id, job_type_to_string(j->type)),
                                    LOG_MESSAGE_ID(SD_MESSAGE_DELETING_JOB_BECAUSE_ORDERING_CYCLE_STR),
+                                   LOG_ITEM("DELETED_UNIT=%s", delete->unit->id),
+                                   LOG_ITEM("DELETED_TYPE=%s", job_type_to_string(delete->type)),
                                    LOG_ITEM("%s", strna(unit_ids)));
 
                         if (log_get_show_color())
